app.rb

```diff
@@ -120,4 +121,16 @@ def list_rentals
       puts ''
     end
   end
+
+  def save_data
+    save_data_to_json('books.json', @books)
+    save_data_to_json('people.json', @person)
+    save_data_to_json('rentals.json', @rentals)
+  end
+
+  private
+
+  def save_data_to_json(filename, data)
+    File.write(filename, JSON.generate(data))
+  end
```

Comment:
- Good job done so far, but you are yet to implement all projects requirements, Part of the project requirements say:
`On startup of your app load all data from files. Remember to handle the case when one or more of the files are missing`. Please fix this 🙏